### ISSUE TITLE
Add support for dnf module install <package>-<version>

### ISSUE THREAD

**Issue by krahb** [hadolint/hadolint#702]
- [x] This is a bug report
- [x] This is a feature request
- [ ] I searched existing issues before opening this one

### Expected behavior
On Fedora/CentOS/RHEL based images the command ```dnf module install nginx:1.18 -y``` should be allowed, but it looks like hadolint does not support the ```dnf module``` command.

### Actual behavior
This error is thrown: ```DL3041 warning: Specify version with `dnf install -y <package>-<version>```

### Steps to reproduce the behavior
Dockerfile:
```
FROM registry.access.redhat.com/ubi8/ubi:latest
RUN dnf module install nginx:1.18 -y && \
    dnf clean all --assumeyes
```
Linter:
``` bash
docker run --rm -i hadolint/hadolint < Dockerfile
-:1 DL3007 warning: Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag
-:2 DL3041 warning: Specify version with `dnf install -y <package>-<version>`.
```
**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version` or
  `docker run --rm ghcr.io/hadolint/hadolint hadolint --version`:**

```bash
Haskell Dockerfile Linter v2.7.0-0-g3e13ec6
```
